### ISSUE TITLE
Warrior Opener with Furious Slash Talent

### ISSUE THREAD

**Issue by InvalidTheory** [herotc/hero-rotation#529]
Currently, Hero Rotation is recommending that we use Furious Slash too early in the opener.  The opener for Warriors using 2133123 Talents (standard single-target talents) along with Condensed Life-Force (standard single-target essence) should be:

Pre-pull:
-Condensed Life-Force Essence
-Recklessness
-Charge

On-pull:
-Siegebreaker
-Rampage
-Bloodthirst
-Dragon Roar
-Raging Blow
-Rampage
-Bloodthirst

... (until Recklessness ends)

-Furious slash
-Bloodthirst
-Rampage

etc etc etc...

Basically, we shouldn't use Furious Slash at all during the opener while Recklessness is active, after which we start stacking and maintaining the Furious Slash haste buff for the rest of the fight.  It is worth noting that Furious Slash _IS_ used during Recklessness during combat (non-opener) if the haste buff will drop off, so this is just an opener issue.  

Thanks!

**Comment by InvalidTheory** [comment]
My mistake, the T25 simc profile (along with Hero Rotation) check for Heroism/Bloodlust, not Recklessness, during the opener...and I wasn't using this in my testing.  Here is the relevant code from the T25 profile:

```
actions.single_target=siegebreaker
actions.single_target+=/rampage,if=(buff.recklessness.up|buff.memory_of_lucid_dreams.up)|(talent.frothing_berserker.enabled|talent.carnage.enabled&(buff.enrage.remains<gcd|rage>90)|talent.massacre.enabled&(buff.enrage.remains<gcd|rage>90))
actions.single_target+=/execute
actions.single_target+=/furious_slash,if=!buff.bloodlust.up&buff.furious_slash.remains<3
actions.single_target+=/bladestorm,if=prev_gcd.1.rampage
actions.single_target+=/bloodthirst,if=buff.enrage.down|azerite.cold_steel_hot_blood.rank>1
actions.single_target+=/dragon_roar,if=buff.enrage.up
actions.single_target+=/raging_blow,if=charges=2
actions.single_target+=/bloodthirst
actions.single_target+=/raging_blow,if=talent.carnage.enabled|(talent.massacre.enabled&rage<80)|(talent.frothing_berserker.enabled&rage<90)
actions.single_target+=/furious_slash,if=talent.furious_slash.enabled
actions.single_target+=/whirlwind
```
